index.html

```diff
@@ -0,0 +1,122 @@
+        >
+          <div class="navbar-nav">
+            <a
+              class="nav-item nav-link active text-white list-page list "
+              href="#"
+              >List <span class="sr-only">(current)</span></a
+            >
+            <a class="nav-item nav-link text-white add-page add" href="#"
+              >Add New</a
+            >
+            <a
+              class="nav-item nav-link text-white contact-page contact-nav"
+              href="#"
+              >Contact</a
+            >
+          </div>
+        </div>
+      </div>
+    </nav>
+    <div id="current-date" class="display-4 text-center mt-5 text-secondary "></div>
+
+  </header>
+
+<body class="body">
+    <div class="container b1 mt-4"  id ="tableList">
+        <h1 class="display-4 text-center">
+          <i class="fas fa-book-open text-primary"></i> My<span class="text-primary">Book</span>List</h1>
+          <form id="book-form">
+            <div class="form-group">
+              <label for="title">Title</label>
+              <input type="text" id="title" class="form-control">
+            </div>
+            <div class="form-group">
+              <label for="author">Author</label>
+              <input type="text" id="author" class="form-control">
+            </div>
+            <div class="form-group">
+              <label for="isbn">ISBN#</label>
+              <input type="text" id="isbn" class="form-control">
+            </div>
+            <input type="submit" value="Add Book" class="btn btn-primary btn-block">
+          </form>
+          <table class="table table-striped mt-5">
+            <thead id ="book-list">
+              <tr>
+                <th>Title</th>
+                <th>Author</th>
+                <th>ISBN#</th>
+                <th></th>
+              </tr>
+            </thead>
+            <tbody id="book-list"></tbody>
+          </table>
+ </div>
+
+    <section id="contact" class="contact-div">
+      <div class="container  d-flex justify-content-center my-5">
+        <div class="contact-details mx-5">
+          <h1 class="display-4 text-center mt-5 text-secondary font-weight-900">
+            Contact Information
+          </h1>
+          <div class="container ">
+            <p>Do you have any questions or you just want say hello?</p>
+            <p>You can reach out to us!</p>
+            <ul>
+              <li>Email: <strong>abc123@gmail.com</strong></li>
+              <li>Phone Number: <strong>+123456789</strong></li>
+              <li>Address: <strong>123 045</strong></li>
+            </ul>
+          </div>
+        </div>
+      </div>
+    </section>
+
+    
+    <script src="./storage.js" type="module"></script>
+    <script src="./UI.js" type="module"></script>
+    <script src="./display.js" type="module"></script>
+    <script src="./index.js" type="module"></script>
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/luxon/2.0.0/luxon.min.js"></script>
```

Comment:
- [ ] Following the project requirement, you are meant to only import the `index.js`  and `index.css` file here.
> Create the main index.html file. Link your index.js and index.css files there.
- [ ] Other js files should be imported in the `index.js` file. :+1: :+1: . I'll suggest you visit your codebase and implement this. :+1: